test/backend/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+set (ONNX_MLIR_BACKEND "onnx-mlir")
```

Comment:
I would rename it to something like `TEST_DRIVER`. The word _backend_ has many meanings in the context of both ONNX and MLIR. Probably don't want to confuse users any futher.

Reply:
Sure, I often struggle with naming things. Changed the name, and just removed a redundant comment in test.py.